.buildkite/scripts/dra_release.sh

```diff
@@ -41,9 +41,7 @@ mkdir -p ${BASE_DIR}/reports
 ./dev-tools/dependencies-report --csv ${BASE_DIR}/reports/dependencies-${VERSION}.csv
 cd ${BASE_DIR}/reports && shasum -a 512 dependencies-${VERSION}.csv > dependencies-${VERSION}.csv.sha512
 
-cd ${WORKSPACE}
+cd $(dirname ${WORKSPACE})
 export FOLDER="${FOLDER_PATH}"
```

Comment:
Could it be change FOLDER here , so it is used as before inside release-manager.sh ? 

```suggestion
export FOLDER="${WORKSPACE}"/"${FOLDER_PATH}"
```

Reply:
Then it becomes BASE_DIR
`BASE_DIR="${WORKSPACE}/${FOLDER_PATH}"`

Reply:
@sharbuz I see also that inside `release_manager` it is used `$PWD`. what is the expected path where that docker command should run ?